gradle/buildsrc.libs.versions.toml

```diff
@@ -1,9 +1,10 @@
 [versions]
-asciidoctorj = '4.0.2'
-asset-pipeline-gradle = '4.4.0'
-grails-gradle-plugin = '6.1.2'
+asciidoctorj = '4.0.3'
+asset-pipeline-gradle = '5.0.0'
+grails-gradle-plugin = '7.0.0-SNAPSHOT'
 webdriver-binaries = '3.2'
-spock = '2.3-groovy-3.0'
+spock = '2.4-M4-groovy-3.0'
+groovy = '4.0.22'
```

Comment:
It was for this for buildSrc/build.gradle

```configurations.configureEach {
    resolutionStrategy.eachDependency { DependencyResolveDetails details ->
        if ((details.requested.group == 'org.codehaus.groovy' || details.requested.group == 'org.apache.groovy') && details.requested.name != 'groovy-bom') {
            String groovyVersion = findProperty('groovyVersion') ?: buildsrcLibs.versions.groovy.get()
            details.useTarget(group: 'org.apache.groovy', name: details.requested.name, version: groovyVersion)
            details.because "The dependency coordinates are changed in Apache Groovy 4, plus ensure version"
        }
    }
}
```

However, since we're sticking with Groovy 3 for Gradle buildscript, I just removed that entirely and also removed 
this `groovy = '4.0.22'` from `gradle/buildsrc.libs.versions.toml`.